README.md

```diff
@@ -79,3 +79,47 @@ Generate mocks like this:
 go install github.com/vektra/mockery/v2@latest
 make mock-generate
 ```
+
+## CLI
+
+Storage example
```

Comment:
maybe level-3 heading?

Reply:
Then it would go under `Installation` chapter. Will it be correct?

Reply:
I just meant a subsection of CLI section for storage examples. When we add other domains, they will be in their own subsections.

Reply:
Updated